### ISSUE TITLE
Storing learning related parameters for checkpoints

### ISSUE THREAD

**Issue by erogol** [apache/incubator-mxnet#1600]
For any pre-trained network, if you don't know the latest learning rate used then it is pain to fine-tune it with a good initial learning rate. therefore, I believe it is good to keep learning parameters with the checkpoint models for the continuation of training later.

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
